stack.cabal

```diff
@@ -187,7 +187,7 @@ library
                    , hit
                    , hpc
                    , http-client >= 0.5.0
```

Comment:
Strictly speaking, shouldn't d52c9bcdd2bf717b8278d90a1cdc4ff68232545e also increase the cabal bound to `http-client >= 0.5.3.1`?